modules/mobile/app/controllers/mobile/v0/facility_eligibility_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class FacilityEligibilityController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 3
+      SERVICE_TYPES = %w[primaryCare nutrition podiatry optometry audiology].freeze
+
+      def index
+        require_params
+        unless SERVICE_TYPES.include?(params[:serviceType])
+          raise Common::Exceptions::InvalidFieldValue.new('serviceType', params[:serviceType])
+        end
+
+        page_facilities, page_meta_data = paginate(params[:facilityIds])
```

Comment:
Took @kpethtel pagination code used on the cc provider, will create a ticket that can be worked once all the appointment requests endpoints are complete to put all pagination under one library.

Reply:
Created: https://app.zenhub.com/workspaces/va-mobile---backend-61d75d03a7505500147ec7f6/issues/department-of-veterans-affairs/va-mobile-app/3086

Reply:
Thanks!